factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgMetrics.java

```diff
@@ -80,7 +81,7 @@ public void time(@NonNull OP operation, @NonNull Runnable r) {
     }
   }
 
-  public void time(@NonNull OP operation, @NonNull Sample sample, Exception e) {
+  private void time(@NonNull OP operation, @NonNull Sample sample, Exception e) {
```

Comment:
shouldn't this one stay public?

Reply:
I don't see the use case to pass the start time for an operation. I only see this for distributed operations.